apps/web/src/app/(signing)/sign/[token]/complete/page.tsx

```diff
@@ -95,8 +97,7 @@ export default async function CompletedSigningPage({
     fields.find((field) => field.type === FieldType.NAME)?.customText ||
     recipient.email;
 
-  const sessionData = await getServerSession();
```

Comment:
I removed this because it's repeating and the only use case was to check for logged in users